pkgs/misc/drivers/logiops/default.nix

```diff
@@ -1,20 +1,30 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, udev, libevdev, libconfig }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, glib, udev, libevdev
+, libconfig }:
 
 stdenv.mkDerivation rec {
   pname = "logiops";
-  version = "0.2.3";
+  version = "0.3.3";
 
   src = fetchFromGitHub {
-    owner = "pixlone";
+    owner = "PixlOne";
     repo = "logiops";
     rev = "v${version}";
-    sha256 = "sha256-1v728hbIM2ODtB+r6SYzItczRJCsbuTvhYD2OUM1+/E=";
+    sha256 = "sha256-9nFTud5szQN8jpG0e/Bkp+I9ELldfo66SdfVCUTuekg=";
+    fetchSubmodules = true;
   };
 
-  PKG_CONFIG_SYSTEMD_SYSTEMDSYSTEMUNITDIR = "${placeholder "out"}/lib/systemd/system";
-
```

Comment:
Is there a reason you got rid of the systemd unit? It can just not be included in `systemd.packages` if it is not desirable.

Reply:
I thought that it would make more sense to split the installation of the `logid` binary from the installation of the `systemd` service, to let the user have more flexibility. Additionally, defining the `systemd` service separately would allow us to configure it with more hardened security privileges I believe(?)

Reply:
Yes, but it is already split by `environment.systemPackages` not implying `systemd.packages`, so the unit in the package will not be defined. It's nice to ship what upstream ships in our package, if someone wants it. And more importantly: `patches` are fragile.

Reply:
Ah I see, so [according to this](https://nixos.org/manual/nixos/stable/#sect-nixos-systemd-nixos), even if we allow the CMake to "install" the service, it won't actually do anything(?) Thanks for your patience.

Reply:
Yup